XRTK-Core/Packages/com.xrtk.core/Definitions/DiagnosticsSystem/MixedRealityDiagnosticsProfile.cs

```diff
@@ -2,89 +2,81 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using UnityEngine;
-using XRTK.Attributes;
+using UnityEngine.Serialization;
 using XRTK.Definitions.Utilities;
-using XRTK.Interfaces.Diagnostics;
 
 namespace XRTK.Definitions.Diagnostics
 {
     /// <summary>
     /// Configuration profile settings for setting up diagnostics.
     /// </summary>
-    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Diagnostics System Profile", fileName = "MixedRealityDiagnosticsProfile", order = (int)CreateProfileMenuItemIndices.Diagnostics)]
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Diagnostics Profile", fileName = "MixedRealityDiagnosticsProfile", order = (int)CreateProfileMenuItemIndices.Diagnostics)]
```

Comment:
We can drop the Mixed Reality prefix here I think

Reply:
Will do. Didn't see that one!

Reply:
Updated